### PR TITLE
[config] Add config template and make it compatible with tikv conf

### PR DESCRIPTION
**Pull request by cfzjywxk**
Issue #300   
Add `raftstore` related configs and make them compatible with tikv,  these will be used by jepsen to make it fail fast

### CODE REVIEW

**Comment by cfzjywxk** [comment]
@coocood @bobotu PTAL

File: config/config.go [hunk 1]
```diff
@@ -35,14 +48,23 @@ type Engine struct {
...
+		RegionSize: 64 * MB,
+		LogLevel:   "info",
+		MaxProcs:   0,
+		Raft:       true,
+	},
+	RaftStore: RaftStore{
+		RaftWorkers:              2,
+		PdHeartbeatTickInterval:  20,
+		RaftStoreMaxLeaderLease:  9,
+		RaftBaseTickInterval:     1,
```
Comment (coocood) [inline config/config.go]: This default value is too small.

File: etc/config-template.toml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+## Unistore config template
```
Comment (coocood) [inline etc/config-template.toml]: I think it's better to put it into the `config` package.

File: config/config.go [hunk 3]
```diff
@@ -35,14 +48,23 @@ type Engine struct {
...
+		StoreAddr:  "127.0.0.1:9191",
+		StatusAddr: "127.0.0.1:9291",
+		RegionSize: 64 * MB,
+		LogLevel:   "info",
+		MaxProcs:   0,
+		Raft:       true,
+	},
+	RaftStore: RaftStore{
+		RaftWorkers:              2,
+		PdHeartbeatTickInterval:  20,   // 20s
```
Comment (coocood) [inline config/config.go]: We better use the consistent unit, milliseconds.
Or make it clear in the name.

**Comment by coocood** [comment]
LGTM

**Pull request merged**